moped-editor/src/components/GridTable/Filters.js

```diff
@@ -118,17 +118,26 @@ const generateEmptyField = (uuid) => {
 
 /**
  * Filter Search Component aka Advanced Search
- * @param {Object} query - The main query object
- * @param {Object} filterState - The current state/state-modifier bundle for filters
+ * @param {Object} filters - The current filters from useAdvancedSearch hook
+ * @param {Function} setFilters - Set the current filters from useAdvancedSearch hook
+ * @param {Object} filterQuery - The current filter query string from useAdvancedSearch hook
+ * @param {Object} history - The history to update query string parameters
+ * @param {Function} handleAdvancedSearchClose - Used to close the advanced search
+ * @param {Object} filtersConfig - The configuration object for the filters
+ * @param {Function} setSearchFieldValue - Used to set the search field value
+ * @param {Function} setSearchTerm - Used to set the search term for simple search
  * @return {JSX.Element}
  * @constructor
  */
-const GridTableFilters = ({
-  query,
-  filterState,
```

Comment:
One of the bigger changes that I made in this component was breaking `filterState` apart since it was an object that contained both the state set function and the state for filters. It also renamed them. I ended up passing those around separately using their original names.